test/integration/models/test_linode.py

```diff
@@ -0,0 +1,428 @@
+
+    assert linode.status == "resizing"
+
+    # Takes about 3-5 minute to resize, sometimes longer...
+    wait_for_condition(30, 600, get_status, linode, "running")
+
+    assert linode.status == "running"
+
+
+def test_linode_resize_with_class(
+    get_client, create_linode_for_long_running_tests
+):
+    linode = create_linode_for_long_running_tests
+    ltype = Type(get_client, "g6-standard-4")
+
+    wait_for_condition(10, 100, get_status, linode, "running")
+
+    num_of_tries = 0
+
+    while num_of_tries <= 3:
+        try:
+            linode.resize(new_type=ltype)
+        except ApiError:
+            if num_of_tries > 3:
+                raise ApiError("number of tries exceeded")
+        num_of_tries += 1
+
+    wait_for_condition(10, 100, get_status, linode, "resizing")
+
+    assert linode.status == "resizing"
+
+    # Takes about 3-5 minute to resize, sometimes longer...
+    wait_for_condition(30, 600, get_status, linode, "running")
+
+    assert linode.status == "running"
+
+
+def test_linode_boot_with_config(create_linode):
+    linode = create_linode
+
+    wait_for_condition(10, 100, get_status, linode, "running")
+    retry_sending_request(3, linode.shutdown)
+
+    wait_for_condition(30, 300, get_status, linode, "offline")
+
+    config = linode.configs[0]
+
+    retry_sending_request(3, linode.boot, config)
+
+    wait_for_condition(10, 100, get_status, linode, "running")
+
+    assert linode.status == "running"
+
+
+def test_linode_firewalls(create_linode_with_volume_firewall):
+    linode = create_linode_with_volume_firewall
+
+    firewalls = linode.firewalls()
+
+    assert len(firewalls) > 0
+    assert "TestSDK" in firewalls[0].label
+
+
+def test_linode_volumes(create_linode_with_volume_firewall):
+    linode = create_linode_with_volume_firewall
+
+    volumes = linode.volumes()
+
+    assert len(volumes) > 0
+    assert "TestSDK" in volumes[0].label
+
+
+def test_linode_disk_duplicate(get_client, create_linode):
+    pytest.skip("Need to find out the space sizing when duplicating disks")
+    linode = create_linode
+
+    disk = get_client.load(Disk, linode.disks[0].id, linode.id)
+
+    try:
+        dup_disk = disk.duplicate()
+        assert dup_disk.linode_id == linode.id
+    except ApiError as e:
+        assert e.status == 400
+        assert "Insufficient space" in str(e.json)
+
+
+def test_linode_instance_password(create_linode_for_pass_reset):
+    linode = create_linode_for_pass_reset[0]
+    password = create_linode_for_pass_reset[1]
+
+    wait_for_condition(10, 100, get_status, linode, "running")
+
+    retry_sending_request(3, linode.shutdown)
+
+    wait_for_condition(10, 200, get_status, linode, "offline")
+
+    linode.reset_instance_root_password(root_password=password)
+
+    linode.boot()
+
+    wait_for_condition(10, 100, get_status, linode, "running")
+
+    assert linode.status == "running"
+
+
+def test_linode_ips(create_linode):
+    linode = create_linode
+
+    ips = linode.ips
```

Comment:
I think this is failing due to the issue resolved in [this PR](https://github.com/linode/linode_api4-python/pull/280). I'll put up a PR to backport this change to `dev` and you should be able to pull it in 👍